doc/build/faq/ormconfiguration.rst

```diff
@@ -321,10 +321,10 @@ won't see that anything actually went wrong.
 
 The solution to this problem is to always specify a deterministic sort order,
 so that the main query always returns the same set of rows. This generally
-means that you should :meth:`_query.Query.order_by` on a unique column on the table.
+means that you should :meth:`_sql.Select.order_by` on a unique column on the table.
 The primary key is a good choice for this::
 
-    session.query(User).options(subqueryload(User.addresses)).order_by(User.id).first()
+    session.scalars(select(User).options(subqueryload(User.addresses)).order_by(User.id)).first()
```

Comment:
**mike bayer** (zzzeek) wrote:

limit(1)

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3729

Reply:
Changed